koku/koku/configurator.py

```diff
@@ -534,6 +556,18 @@ def get_database_ca_file():
             return LoadedConfig.rds_ca()
         return None
 
+    @staticmethod
+    def _use_read_replica():
+        read_replica_file_list = [
+            "/etc/db/readreplica/db_host",
+            "/etc/db/readreplica/db_port",
+            "/etc/db/readreplica/db_name",
+            "/etc/db/readreplica/db_user",
+            "/etc/db/readreplica/db_password",
+        ]
+        use_read_replica = ENVIRONMENT.bool("USE_READREPLICA", default=False)
+        return use_read_replica and all(list(map(os.path.isfile, read_replica_file_list)))
```

Comment:
`all` accepts an iterable, so there's no need to unwind the iterable from `map` into a list.
```suggestion
        return use_read_replica and all(map(os.path.isfile, read_replica_file_list))
```

A generator expression can be used instead of `map()`.
```suggestion
        return use_read_replica and all(pathilb.Path(file).is_file() for file in read_replica_file_list)
```

Reply:
updates made.